### DISCUSSION TITLE
Get the text matched on GestureRecognizer onTap

### DISCUSSION THREAD

**Discussion by Paroca72** [2000calories/flutter_easy_rich_text#48] [General]
Hello,

```
final EasyRichTextPattern missing = EasyRichTextPattern(
    targetString: r"#\b\w+\b",
    matchWordBoundaries: false,
    style: const TextStyle(
      color: ThemesManager.templateErrorColor,
      fontWeight: FontWeight.bold,
    ),
    recognizer: TapGestureRecognizer()..onTap = () {
      // Word I matched
    },
  );
```

There is a way to know the word on I tapped?
Using GestureRecognizer I not find a way.. so I suppose will be very useful to have a custom event onTap with, as parameter, the word tapped.

Thanks for your help
Sam

**Comment by 2000calories** [comment]
You can use `matchBuilder`. Check the following reference.
https://github.com/2000calories/flutter_easy_rich_text/issues/24#issuecomment-963216279
